### DISCUSSION TITLE
Temporally varying resistivity

### DISCUSSION THREAD

**Discussion by veraberger** [PrincetonUniversity/athena#598] [Q&A] [answered]
Hi, I've used the EnrollFieldDiffusivity function to successfully implement a spatially varying resistivity function in my MHD simulations, but is it possible to also vary the resistivity in time?

**Comment by tomidakn** [comment, marked as answer]
This function is called every timestep, so it can be time dependent. If you need time explicitly, you can access it through pmb->pmy_mesh->time.

**Comment by veraberger** [comment]
Thank you very much, and for such a quick response!
